client/src/components/Page/ChooseYourRunPage.js

```diff
@@ -28,32 +28,22 @@ export default function ChooseYourRunPage() {
       <ChooseYourRunPageContainer>
         <AppHeader title={"Choose Your Run"} />
         <ScrollContainer>
-          <Link to="/created_run">
-            <RunCard
-              date={"44.44.444"}
-              time={"14:44"}
-              distance={"10"}
-              isFavorite={true}
-              runName={"Severin Brücke Run"}
-            ></RunCard>
-            <RunCard
-              date={"44.44.444"}
-              time={"14:44"}
-              distance={"10"}
-              runName={"Grüngürtel"}
-            ></RunCard>
-            <RunCard
-              date={"44.44.444"}
-              time={"14:44"}
-              distance={"10"}
-            ></RunCard>
+          <RunCard
+            date={"44.44.444"}
+            time={"14:44"}
+            distance={"10"}
+            isFavorite={true}
+            runName={"Severin Brücke Run"}
+          ></RunCard>
```

Comment:
You can use short form here (multiple times)
```suggestion
          />
```

Reply:
Thanks